LFS171x/fabric-material/first-network/README.md

```diff
@@ -0,0 +1,6 @@
+## Build Your First Network (BYFN)
```

Comment:
Files in the first-network folder are no longer needed. We are pulling these from the fabric-samples repository with git clone https://github.com/hyperledger/fabric-samples.git

Reply:
ahh yes I'll delete this folder